### PR TITLE
migrate test for read-only Java editor mode from Selenium

### PR DESCRIPTION
**Pull request by barboras7**
Reimplementation of https://github.com/droolsjbpm/kie-wb-common/pull/364

### CODE REVIEW

**Comment by barboras7** [comment]
jenkins retest this

**Comment by manstis** [comment]
@wmedvede Could you please review this for @barboras7 thanks.

File: kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-client/src/test/java/org/kie/workbench/common/screens/datamodeller/client/SourceEditorReadOnlyTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    @Mock
+    private User user;
+
+    @Before
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+
+        //startUp() related setup
```
Comment (wmedvede) [inline kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-client/src/test/java/org/kie/workbench/common/screens/datamodeller/client/SourceEditorReadOnlyTest.java]: Seems like this line is out-dated?

**Comment by barboras7** [comment]
@wmedvede yes, I forgot to remove that comment, it is fixed now

**Comment by barboras7** [comment]
jenkins retest this

**Comment by wmedvede** [comment]
+1, thanks

**Pull request merged**